pkg/vpcmodel/explainabilityInput.go

```diff
@@ -344,7 +345,7 @@ func (c *VPCConfig) getNodesOfVsi(name string) ([]Node, int, error) {
 		vpc = cidrOrNameSlice[0]
 		vsi = cidrOrNameSlice[1]
 	}
-	for _, nodeSet := range c.NodeSets {
+	for _, nodeSet := range append(c.NodeSets, c.loadBalancersAsNodeSets()...) {
```

Comment:
the function name is now misleading (`getNodesOfVsi`), it returns also LB nodes, not only VSI nodes

Reply:
yes, the function is not just misleading, it is also wrong:
1. it returns also vpe nodes
2. it is part of the vpcmodel, a package that should not know what a vsi, or vpe is

the task of separating the ibm specific references from vpcmodel package is out of the scope of this PR.

Reply:
The (original) function returns Nodes representing an endpoint given by name. It is not ibm specific and has the correct abstraction level. Indeed, the names (of the func and of the variables) are misleading. The function is not wrong and no need for a PR in the regard.

Reply:
current code and documentation *was* not consistent and misleading.
1. If I will change the function name, to context of the function call will be misleading
2. all what you ask here is out of the scope of this PR. 
3. I added a todo comment, you are welcome to follow it

Reply:
@ShiriMoran , will open an issue for you to revisit this function.

Reply:
The function is consistent; some documentation and renaming could indeed help.